### ISSUE TITLE
Cannot login to specific server - character duplicated

### ISSUE THREAD

**Issue by bellisco** [StrangeLoopGames/EcoIssues#6668]
Detailed Description of Issue:
Was mining when i was kicked by "inactivity". I tried to rejoin, but was stuck on "loading world". I've since tried to rejoin with no results. After about an hour i got in, but my character was duplicated and responded to my actual character input. Video of the incident here: [https://youtu.be/h50xnY9Llgg](url)
While i was joining and stuck on loading world, i was a few times able to load the UI with no world (the environment was black, but i could switch tools. I've attached the output files to this log. The first is from not being able to join
[output_log.txt](https://github.com/StrangeLoopGames/EcoIssues/files/1860645/output_log.txt)

The second is from the session where i was finally able to join, but my character was duplicated
[output_log_duplication.txt](https://github.com/StrangeLoopGames/EcoIssues/files/1860663/output_log_duplication.txt)

Reproducible? How difficult is it to reproduce?
Not being able to join is not quite hard FOR ME. Being able to join i cannot attest to, as i was only able to rejoin once with the duplicated character

Steps to reproduce bug/issue:
N/A

Additional Comments:

Server, Client or Website?
[{"IP":"31.214.224.88","WebPort":3451,"Description":"World of EcoCraft"}]

OS/Web Browser?
Windows 10 Professional

Client Version? [Alpha .01 for example]
7.3.0

**Comment by MarjieAVolk** [comment]
I saw this unusual exception in both of your output logs:

```
Caught exception on WorldLayerManager(Clone) while in NetObject ReceiveInitialState.
Exception: Expected to receive type key, since there is no view mapping for GUID 240625, but type key was not given
Callstack:   at Eco.Shared.View.ViewManager.GetView (Eco.Shared.Serialization.BSONObject bson) [0x00000] in <filename unknown>:0 
  at Eco.Shared.View.ViewManager.UnpackageViewInternal (Eco.Shared.Serialization.BSONObject bson, System.Collections.Generic.List`1 notifications) [0x00000] in <filename unknown>:0 
  at Eco.Shared.View.ViewManager.UnpackageView[ClimateStateView] (Eco.Shared.Serialization.BSONObject bson) [0x00000] in <filename unknown>:0 
  at WorldLayerManager.ReceiveInitialState (Eco.Shared.Serialization.BSONObject bson) [0x00000] in <filename unknown>:0 
  at NetObjComponent.OnReceiveInitialState (Eco.Shared.Serialization.BSONObject bsonObj) [0x00000] in <filename unknown>:0 
  at NetObjComponent.Eco.Shared.Networking.INetObject.ReceiveInitialState (Eco.Shared.Serialization.BSONObject bsonObj) [0x00000] in <filename unknown>:0 
BSON: Eco.Shared.Serialization.BSONObject
```

Well, just some standard debug steps that might help:
* Delete the folder  %APPDATA%\..\LocalLow\Strange Loop Games\ECO
* Use steam's Verify integrity of game files

**Comment by bellisco** [comment]
The game crashed the first time i launched after deleting the folder and verifying integrity. It seems that your fix did work, as i spawned in the position that my duplicated character was in

**Comment by MarjieAVolk** [comment]
O.o well, I guess I will count this as a success :P

**Comment by metachronism** [comment]
99% sure this is fixed with the 7.3.1 hotfix

**Comment by Quagmire141** [comment]
This issue is absolutely not fixed with 7.3.1.  Had a long day with a server full of players dropping then being stuck on "Entering World..."

Tried the deleting %APPDATA% that does indeed allow me to get back onto the server, multiple 'ghost' copies of myself were around stuck in idle animations including name tag.

**Comment by metachronism** [comment]
possible fix pending for 7.3.2 (hard to repro, feel free to reopen if still happening after 7.3.2 is released)
